backend/src/v5/routes/teamspaces/projects/models/common/tickets.comments.js

```diff
@@ -419,6 +419,204 @@ const establishRoutes = (isFed) => {
 	 *                 items:
 	 *                   type: string
 	 *                   description: Image in a Base64 format
+	 *               views:
```

Comment:
We should update the schema of Get Comment and Get All Comments endpoints since the views are returned as well